homeassistant/components/google_assistant/trait.py

```diff
@@ -251,10 +240,10 @@ def supported(domain, features):
         return domain in (
             group.DOMAIN,
             input_boolean.DOMAIN,
+            cover.DOMAIN,
```

Comment:
Unnecessary change.

Reply:
I don't understand why we should keep covers in the 'OnOff' trait.

Reply:
I don't understand why we should keep covers in the 'OnOff' trait.

Reply:
Its really just a personal preference kind of thing. I saw the code for it was left in, in which case the tests need to remain, and I kind of like the ability to tell google to turn off everything and not  have to switch to close or lock, etc. Can probably be removed if we want to be consistent though and probably should just be handled with the group domain and supported in the cover element of ha.

Reply:
I would agree with @giefca here. If I say turn everything off, one would expect switches and lights to turn off, not curtains to close and locks to lock I think.